src/scss/provider-core.scss

```diff
@@ -1,13 +1,13 @@
 // Core Styles
 // These are the core styles shared between mobile and desktop
-@import 'core/reset';
-@import 'core/base';
-@import 'core/flex';
-@import 'core/fonts';
-@import 'core/icon';
-@import 'core/layout';
-@import 'core/overrides';
-@import 'core/scrollbar';
-@import 'core/space';
-@import 'core/text';
-@import 'core/typography';
+@forward 'core/reset';
```

Comment:
So.. I'm not certain, but what we might be able to do here is:

```scss
@use '../provider-variables' as *;

@use 'core/reset';
...
@use 'core/typography';

@forward 'core/base';
...
@forward 'core/typography';
```

I think that would put them all in the same namespace and the export them all with the shared namespacing..  but I'm not 💯  on that.

I'm also not sure that it's better..

Reply:
Hmm interesting, that's worth investigating. Not sure if that will work either, but would be nice if it did!

Reply:
Couldn't get this to work 😞

Reply:
one other thought that might be a hair cleaner.. at least for now.. we could make a js file at the very top of index called `provider-core-styles.js`.. remove this file and all the `_`d file names.. or something and instead of using sass partials just import all of these at the top..  then they'd get the `additionalData` and we wouldn't have to add that to the partials.